tfjs-layers/src/models_test.ts

```diff
@@ -842,53 +842,51 @@ describeMathCPU('loadLayersModel from URL', () => {
   });
 
   it('loadLayersModel: no memory leak with nested LayersModel layer',
-    async() => {
```

Comment:
Is this related to this change? Curious why this test had to change for this PR?

Reply:
When I ran layers test in local, it gives me the result of 120.99999999, so I change it to closeTo.